astroarch_build.sh

```diff
@@ -118,8 +119,29 @@ systemctl enable x0vncserver
 # Copy the config for kwinrc
 su astronaut -c "cp /home/astronaut/.astroarch/configs/kwinrc /home/astronaut/.config"
 
+# Copy plasma splash
+cp -R /home/astronaut/.astroarch/configs/org.kde.astroarch.desktop /usr/share/plasma/look-and-feel/
+su astronaut -c "cp /home/astronaut/.astroarch/configs/ksplashrc /home/astronaut/.config"
+
+# GPS copy gps config & configure chrony
+rm /etc/default/gpsd
+cp /home/astronaut/.astroarch/configs/gpsd /etc/default/
+echo 'refclock SHM 0 offset 0.5 delay 0.2 refid NMEA' >> /etc/chrony.conf
+echo 'driftfile /var/lib/chrony/drift' >> /etc/chrony.conf
+
+# Configuration Very Secure FTP Daemon
+rm  /etc/vsftpd.conf
```

Comment:
ftp, while known and famous, is unlikely used by astrophotographers. I am fine with vsftpd but it should not be part of the base image, but rather documented in case someone wants to use it